resources/views/empty.blade.php

```diff
@@ -0,0 +1,36 @@
+@extends('layouts.home')
```

Comment:
Where are you getting the layouts.home view ??

Reply:
Its clear you didn't understand this specific sub-task . Reach out to Ssaava for more explanation.

Reply:
Kindly refer to resources/view/home/index.blade.php and see how the page structure is built then implement the same in this view.